tracer/src/Datadog.Trace/Iast/SourceConverter.cs

```diff
@@ -10,6 +10,7 @@
 using Datadog.Trace.Iast.SensitiveData;
 using Datadog.Trace.Iast.Settings;
 using Datadog.Trace.Vendors.Newtonsoft.Json;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
 using Datadog.Trace.Vendors.Newtonsoft.Json.Utilities;
 
```

Comment:
Nit: Add #nullable enable and check for possible null values

Reply:
It's already in line 6

Reply:
My fault!